src/widget/wtrackmenu.h

```diff
@@ -55,6 +55,7 @@ class WTrackMenu : public QMenu {
     void popup(const QPoint& pos, QAction* at = nullptr);
 
   signals:
+    // This signal must be handled by the client of this class to make "LoadTo" work.
```

Comment:
Instead of commenting here, put this above where the LoadTo flag is defined in the Filter enum:
"The loadTrackToPlayer signal emitted from this class must be handled to make LoadTo work."